apps/wallet-mobile/src/NftDetails/NftDetails.tsx

```diff
@@ -88,34 +81,18 @@ export const NftDetails = () => {
   )
 }
 
-const UnModeratedNftImage = ({nft}: {nft: Balance.TokenInfo}) => {
-  const styles = useStyles()
-  const navigateTo = useNavigateTo()
-  return (
-    <TouchableOpacity onPress={() => navigateTo.nftZoom(nft.id)} style={styles.imageWrapper}>
-      <NftPreview nft={nft} style={styles.image} height={IMAGE_HEIGHT} width={IMAGE_WIDTH} contentFit="contain" />
-    </TouchableOpacity>
-  )
-}
+const imageHeight = 380
+const imagePadding = 16
 
-const ModeratedNftImage = ({nft}: {nft: Balance.TokenInfo}) => {
+const SelectableMedia = ({info}: {info: Portfolio.Token.Info}) => {
   const styles = useStyles()
-  const wallet = useSelectedWallet()
   const navigateTo = useNavigateTo()
-  const {status} = useModeratedNftImage({wallet, fingerprint: nft.fingerprint})
-  const canShowNft = status === 'approved' || status === 'consent'
-
-  if (!canShowNft) {
-    return (
-      <View style={styles.imageWrapper}>
-        <NftPreview nft={nft} style={styles.image} height={IMAGE_HEIGHT} width={IMAGE_WIDTH} showPlaceholder />
-      </View>
-    )
-  }
+  const dimensions = useWindowDimensions()
+  const imageWidth = dimensions.width - imagePadding * 2
```

Comment:
why 2?

Reply:
yap, it was already there, is for the horizontal padding (left, right), I'll add a comment.